vector/src/main/java/im/vector/app/core/error/ErrorFormatter.kt

```diff
@@ -106,19 +103,19 @@ class DefaultErrorFormatter @Inject constructor(
                     throwable.error.code == MatrixError.M_THREEPID_AUTH_FAILED -> {
                         stringProvider.getString(R.string.error_threepid_auth_failed)
                     }
-                    throwable.error.code == MatrixError.M_PASSWORD_TOO_SHORT      -> {
+                    throwable.error.code == MatrixError.M_PASSWORD_TOO_SHORT -> {
```

Comment:
following reformat are in the tchap side

Reply:
You should apply same thing on matrix sdk

Reply:
No I post the comment to confirm this is ok, these cases are in Tchap (we display custom string according to the thrown error)